packages/editor/src/components/rich-text/style.scss

```diff
@@ -48,54 +48,29 @@
 		}
 	}
 
-	// Style TinyMCE inline boundaries on select inline text elements.
-	&:focus {
```

Comment:
@aduth That's true (re: boundaries don't disappear). This reason seems to be that I removed `:focus` here.

Reply:
Ideally we should remove the selection state, but maybe better to look at it separately.

Reply:
Restored.